src/hooks/lastBlock/useValidationData.tsx

```diff
@@ -0,0 +1,44 @@
+import { PersistedValidationData } from '@polkadot/types/interfaces';
+import { useEffect, useState } from 'react';
+import { usePolkadotJsContext } from '../polkadotJs/usePolkadotJs';
+
+export const validationDataDataType =
+  'Option<PolkadotPrimitivesV1PersistedValidationData>';
+
+export const useValidationData = () => {
+  const { apiInstance, loading } = usePolkadotJsContext();
+
+  const [validationData, setValidationData] =
+    useState<PersistedValidationData | null>();
+
+  useEffect(() => {
+    if (!apiInstance) {
+      return;
+    }
+
+    if (loading) {
+      return;
+    }
+
+    const getValidationData = async () => {
+      const rawValidationData =
```

Comment:
here we should fetch the validation data for the blockhash received in the relaychain last block, as described in the issue (#81)